packages/element/README.md

```diff
@@ -138,6 +138,39 @@ _Returns_
 
 -   `WPElement`: Element.
 
+<a name="createInterpolateElement" href="#createInterpolateElement">#</a> **createInterpolateElement**
```

Comment:
How about we expose it as an experimental function and see how it goes before we commit to the final shape?

Reply:
Hmm, ya that sounds like a good approach.  See 46c464e.